### ISSUE TITLE
`map().open()` gets nodes mixed up

### ISSUE THREAD

**Issue by eallik** [amark/gun#516]
<img width="947" alt="screen shot 2018-03-12 at 14 33 44" src="https://user-images.githubusercontent.com/17533/37283966-89e46524-2602-11e8-9ed4-fcb1d729e648.png">

**Comment by eallik** [comment]
As a workaround, I could try `map().on(... and then `.open()` here ...)`, but in that case, I'd have to keep track of which nodes already are `open` and which aren't, which brings us to what I mentioned in my comment to https://github.com/amark/gun/issues/458

**Comment by amark** [comment]
@eallik you are right, it seems to be returning the internal meta object that keeps tracks of `open` state. I wonder if it has to do with the it happening because of one of the things as a string. Will look into this later. Thanks for the reports!
